File: java_backup/my java/souvaghya/fibo.java
class fibo
{
public static void main(int n)
{
int a,b,i,c;
a=0;
b=1;
System.out.print(a+","+b);
for(i=1;i<=n-2;i++)
{
c=a+b;
System.out.print(","+c);
a=b;
b=c;
}
}
}

File: 6th_sem/Systemfiles_Anubhab/FINALMATLAB/MATLABCIA/MATLAB/fibo.m
%fibo.m: Write a program to print fibonacci series of n terms.

clc;
close all;
clear all;
n=input('\nEnter number of terms of fibonacci series(1-10): ');
for i = 1:n
    y=fib(i);
    fprintf('%d\t',y);
end
fprintf('\n');
%End of program